### ISSUE TITLE
Reviewing a PR

### ISSUE THREAD

**Issue by p-pxmpo** [ubsuny/CP1-24-HW2#55]
Am i supposed to just pick whoever's PR I review or does it get assigned to me?

**Comment by avgagliardo** [comment]
it will be assigned to you by the maintainer. If you don't have one yet you could ask for the next assignment.

**Comment by ojha-aditya** [comment]
I guess since we are so close to the project deadline, it is wise to now start calling dibs on the PRs which are pending approval directly.

**Comment by abruns123** [comment]
Yeah, at this point, if something still needs a pull request, it should be made an issue for some discussion on who'll review it to make sure it gets done.
